yarn.lock

```diff
@@ -600,6 +618,21 @@ ajv@^6.10.0, ajv@^6.12.4:
     json-schema-traverse "^0.4.1"
     uri-js "^4.2.2"
 
+anchor-link@^3.5.1:
```

Comment:
Hrm, why does the Account Kit need anchor-link?

Might be worth cleaning up the Account Kit dependencies before merging it in here to the Session Kit, so we don't have much in the way of extra dependencies.

Ideally - the only thing the Account Kit should add as a dependency is itself.